common/test/acceptance/pages/lms/learner_profile.py

```diff
@@ -0,0 +1,147 @@
+
+    @property
+    def privacy(self):
+        """
+        Get user profile privacy.
+
+        Returns:
+            'all_users' or 'private'
+        """
+        return 'all_users' if self.q(css=PROFILE_VISIBILITY_SELECTOR.format('all_users')).selected else 'private'
+
+    @privacy.setter
+    def privacy(self, privacy):
+        """
+        Set user profile privacy.
+        """
+        self.wait_for_element_visibility('select#u-field-select-account_privacy', 'Privacy dropdown is visiblie')
+
+        if privacy != self.privacy:
+            self.q(css=PROFILE_VISIBILITY_SELECTOR.format(privacy)).first.click()
+            EmptyPromise(lambda: privacy == self.privacy, 'Privacy is set to {}'.format(privacy)).fulfill()
+            self.wait_for_ajax()
+
+            if privacy == 'all_users':
+                self.wait_for_public_fields()
+
+    def field_is_visible(self, field_id):
+        """
+        Check if a field with id set to `field_id` is shown.
+
+        Args:
+            field_id (str): field id
+
+        Returns:
+            True/False
+        """
+        self.wait_for_ajax()
+        return self.q(css='.u-field-{}'.format(field_id)).visible
+
+    def field_is_editable(self, field_id):
+        """
+        Check if a field with id set to `field_id` is editable.
+
+        Args:
+            field_id (str): field id
+
+        Returns:
+            True/False
+        """
+        self.wait_for_ajax()
+        self.make_field_editable(field_id)
+        return self.mode_for_field(field_id) == 'edit'
+
+    @property
+    def visible_fields(self):
+        """
+        Return list of visible fields.
+        """
+        self.wait_for_ajax()
+        self.wait_for_element_visibility('.u-field-username', 'username is not visible')
+
+        fields = ['username', 'country', 'language', 'bio']
+        return [field for field in fields if self.field_is_visible(field)]
+
+    @property
+    def editable_fields(self):
+        """
+        Return list of editable fields currently shown on page.
+        """
+        self.wait_for_ajax()
+        self.wait_for_element_visibility('.u-field-username', 'username is not visible')
+
+        fields = ['country', 'language', 'bio']
+        return [field for field in fields if self.field_is_editable(field)]
+
+    @property
+    def privacy_field_visible(self):
+        """
+        Check if profile visibility selector is shown or not.
+
+        Returns:
+            True/False
+        """
+        self.wait_for_ajax()
+        return self.q(css='#u-field-select-account_privacy').visible
+
+    def country(self, value=None):
+        """
+        Get or set language.
+        """
+        self.value_for_dropdown_field('country', value)
+
+    def language(self, value=None):
+        """
+        Get or set country.
+        """
+        self.value_for_dropdown_field('language', value)
+
+    def aboutme(self, value=None):
```

Comment:
Never mind. I see that the UX calls this "About Me" now. Nit: I'd probably call it ``about_me``.